### DISCUSSION TITLE
Computer recommendation

### DISCUSSION THREAD

**Discussion by alexproteomics** [vdemichev/DiaNN#309] [General]
Hi,
I have a $10K-15K budget to purchase a machine to process my diaPASEF data. Obviously I want a machine that can process the data as fast as possible. I was wondering if someone can give some recommendations. 
Thanks
Alex

**Comment by vdemichev** [comment]
Hi Alex,

The way I prefer it:
- better 2-3 less powerful machines than one very powerful
- go for at least 16 physical cores (e.g. AMD 5950X, Intel 10980XE) and 128Gb RAM

Best,
Vadim

**Reply by alexproteomics** [reply]
Hi Vadim,

Thanks for your prompt reply. I was wondering if your preference is based on the idea of splitting the files to the different machines to generate .quant files in parallel and then importing all .quant files to a single machine for a combined analysis? Or based on the idea that each analysis (i.e., a project with a few dozen files or so) is assigned to a single machine and you can run such analyses in parallel.
One more question, linux or windows? which one would be more robust and easy streamline for this multi-machine strategy?
thanks
Alex
